server/db/seed.test.js

```diff
@@ -0,0 +1,37 @@
+const { db } = require('./');
+const app = require('../app');
+const request = require('supertest');
+
+// console.log('app loaded', app);
+
+// test('Mooooooo', () => {
+//   expect(1 + 1).toBe(2)
+// });
+
+describe('Test the API server', () => {
+    // test('should pass integration test', () => {
+    //   request(app)
+    //     .get('/')
+    //     .expect(200, 'moo')
+    //     .end((err) => {
+    //       if (err) throw done(err);
+    //       done();
+    //     });
+    // })
+
+    test('should pass integration tests', () => {
+        // sample data:
```

Comment:
Hey, the description should be more specific to what test is should pass.  Saying "It should pass integration test" is broad. 

What are you testing?

>

Reply:
Good point, the test description is vague. The test checks whether the GET route of `/api/events/:id` fetches info for an event from the database, based on the event's ID.

I will rewrite the test description.